app/views/api_docs/reference/_properties_list.html.erb

```diff
@@ -29,7 +29,7 @@
         <% end %>
 
         <% if property.attributes.description %>
-          <div class="app-styled-content govuk-!-margin-top-4">
+          <div class="govuk-!-margin-top-4">
```

Comment:
Looking at the API docs, I think you can get away with not having this extra `div` with the margin utility class on, but instead add padding bottom to `.app-api-metadata` instead. Saves having two nested `div`s.